app/javascript/react/containers/AirportShowContainer.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react'
+
+import AirportShowTile from '../components/AirportShowTile'
+
+class AirportShowContainer extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      airport: {id: '', name: '', airport_code: '', location: '', description: ''}
+    }
+  }
+
+  componentDidMount() {
+    let airport_id = this.props.params.id
+    fetch(`/api/v1/airports/${airport_id}.json`)
+    .then(response => {
+      if (response.ok) {
+        return response;
+      } else {
+        let errorMessage = `${response.status} (${response.statusText})`,
+            error = new Error(errorMessage);
+        throw(error);
+      }
+    })
+    .then(response => response.text())
```

Comment:
Please simplify by converting to json in this line, then deleting line 27 (see my response to your QQQ)